vehicles.lua

```diff
@@ -1562,9 +1565,24 @@ end
 -- Get the turning radius of the vehicle and its implements (copied from AIDriveStrategyStraight.updateTurnData())
 function AIDriverUtil.getTurningRadius(vehicle)
 	courseplay.debugVehicle(courseplay.DBG_IMPLEMENTS, vehicle, 'Finding turn radius:')
-
-	local radius = vehicle.maxTurningRadius * 1.05 -- TODO: do we really need this buffer?
-	courseplay.debugVehicle(courseplay.DBG_IMPLEMENTS, vehicle, '  maxTurningRadius by Giants is %.1f', vehicle.maxTurningRadius)
+	
+	local radiusMultiplier = 1.05
+	local radius = 5
+	-- default value if all others fail
+	courseplay.debugVehicle(courseplay.DBG_IMPLEMENTS, vehicle, '  turnRadius set to a default %.1f', radius)
+	local TR = ceil(courseplay:calculateTurnRadius(steeringType, wheelBase, rotMax, CPRatio) * radiusMultiplier)
```

Comment:
I don't think any of the arguments of calculateTurnRadius is defined here...

Reply:
You're absolutely right - had planned to delete it but accidentally left it there